cmd/init.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/rcmachado/changelog/chg"
+	"github.com/spf13/cobra"
+)
+
+func newInitCmd(iostreams *IOStreams) *cobra.Command {
+	return &cobra.Command{
+		Use:   "init",
+		Short: "Initializes a new changelog",
+		Long: `Outputs an empty changelog, with preamble and Unreleased version
+
+You can specify a filename using the --output/-o flag.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			compareURL := "https://github.com/rcmachado/changelog/compare/abcdef...HEAD"
```

Comment:
@rcmachado for now, will the generated link be hardcoded? 

One idea is, in the future, to have additional flags in the command to set the base url and the initial version.

Reply:
No, my mistake! We do have a `--compare-url` option under `release` command. I'll add it here. Thanks for catching this!